discovery_engine_core/ai/tokenizer/src/post_tokenizer/mod.rs

```diff
@@ -114,6 +114,7 @@ where
                 .map(|encoding| self.post_tokenize(encoding))
                 .collect()
         });
+        #[allow(clippy::range_plus_one)] // type mismatch
```

Comment:
❓ What do you mean by type mismatch here?

Reply:
clippy wants us to use a `RangeInclusive` instead of a `Range`, but the collected hash map needs `Range` as values.